librouteros/api.py

```diff
@@ -33,7 +33,7 @@ def parseWord(word):
         '''
         _, key, value = word.split('=', 2)
         value = Parser.apiCast(value)
-        return (key, value)
+        return key, value
```

Comment:
IMO minor change which does not influence the code.

Reply:
I'll remove it.